tools/generate-examples.rb

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'fileutils'
+require 'pathname'
+
+GraalVM = Struct.new(:url, :dir) do
+  def name
+    dir
+  end
+
+  def tarball
+    @tarball ||= File.basename(url)
+  end
+end
+
+GraalVM_21_2_0 = GraalVM.new('https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-21.2.0/graalvm-ce-java11-darwin-amd64-21.2.0.tar.gz', 'graalvm-ce-java11-21.2.0')
+GraalVM_22_3_1 = GraalVM.new('https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-22.3.1/graalvm-ce-java17-darwin-amd64-22.3.1.tar.gz', 'graalvm-ce-java17-22.3.1')
+
+GRAAL_VMS = [
```

Comment:
I'd put a comment here that the first one is used as the "latest" or is otherwise treated specially.

Reply:
That code was already there, but I can document it. I haven't actually committed any of the examples with the newer GraalVM yet.